src/components/WindowSideBarButtons.js

```diff
@@ -94,7 +95,7 @@ export class WindowSideBarButtons extends Component {
             icon={(<CanvasIndexIcon />)}
           />
         )}
-        {panels.annotations && (
+        {panels.annotations && (hasAnnotations || hasAnyAnnotations) && (
```

Comment:
Is this extendable to just have enabled? I ask for the mirador-annotations plugin where we rely on this companion window to start creating annotations.

Reply:
I was thinking about changing the  `window.panels` config  to true/false/maybe, but then figured plugins who cared could just wrap `WindowSideBarButtons` and provide their own behavior?